commands/addContributor.ts

```diff
@@ -1,15 +1,44 @@
 import { SlashCommandBuilder } from '@discordjs/builders';
 import { CommandInteraction } from 'discord.js';
-import { isContributor } from '../utils/airTableCalls';
+import { isContributor, createContributor } from '../utils';
+import { createTwitterHandle } from '../utils/twitterHandle';
 
 export const data = new SlashCommandBuilder()
-    .setName('add-contributor')
-    .setDescription('add yourself to the contributor list')
+  .setName('add-contributor')
+  .setDescription('add yourself to the contributor list')
+  .addStringOption(
+    option => option.setRequired(false)
+      .setName('devdao-id')
+      .setDescription('Enter your DevDAO ID (e.g. #2468)'))
+  .addStringOption(
+    option => option.setRequired(false)
+      .setName('twitter')
+      .setDescription('Enter your twitter handle'))
```

Comment:
could we provide the format for a happy input like we do in the `devdao-id` section?

Reply:
Definitely